R/run.R

```diff
@@ -423,9 +423,9 @@ CmdStanProcs <- R6::R6Class(
     },
     is_error_message = function(line) {
       startsWith(line, "Exception:") ||
-      (regexpr(line, "either mistyped or misplaced.") > 0) ||
-      (regexpr(line, "A method must be specified!") > 0) ||
-      (regexpr(line, "is not a valid value for") > 0)
+      (regexpr("either mistyped or misplaced.", line, perl = TRUE) > 0) ||
+      (regexpr("A method must be specified!", line, perl = TRUE) > 0) ||
+      (regexpr("is not a valid value for", line, perl = TRUE) > 0)
```

Comment:
This is a bug in the use of regexpr, the pattern and text were switched. I think this should now fix #69 but have to double check.